cmd/csaf_checker/links.go

```diff
@@ -53,11 +53,11 @@ func (pgs pages) listed(path string, pro *processor) (bool, error) {
 	pro.badDirListings.use()
 
 	if err != nil {
-		pro.badDirListings.add("Fetching %s failed: %v", base, err)
+		pro.badDirListings.add(ErrorType, "Fetching %s failed: %v", base, err)
```

Comment:
Should .add() be replaced with .error() here (and four lines down) as well? (Just like in other files for consistency?)

Reply:
Yes, was an oversight. Corrected now, even if it was only a minor thing.